docs/sources/reference/components/prometheus/prometheus.exporter.unix.md

```diff
@@ -163,6 +165,13 @@ The default values vary by the operating system {{< param "PRODUCT_NAME" >}} run
 ```
 {{< /code >}}
 
+### hwmon block
+
+| Name               | Type       | Description                                                         | Default  | Required |
+|--------------------|------------|---------------------------------------------------------------------|----------|----------|
+| `chip_include`     | `string`   | Regexp of hwmon chip to include mutually exclusive to chip-exclude. |          | no       |
```

Comment:
```suggestion
| `chip_include`     | `string`   | Regexp of hwmon chip to include (mutually exclusive to chip-exclude). |          | no       |
```

Reply:
If you accept the suggestions in github directly, make sure that you reformat the tables. Same for the comment below

Reply:
fixed